docs/resources/Channel.md

```diff
@@ -1086,11 +1095,11 @@ Edit the channel permission overwrites for a user or role in a channel. Only usa
 
 ###### JSON Params
 
-| Field | Type   | Description                                     |
-|-------|--------|-------------------------------------------------|
-| allow | string | the bitwise value of all allowed permissions    |
-| deny  | string | the bitwise value of all disallowed permissions |
-| type  | integer | 0 for a role or 1 for a member                  |
+| Field | Type    | Description                                     | Default |
+|-------|---------|-------------------------------------------------|---------|
+| allow | string  | the bitwise value of all allowed permissions    | 0       |
+| deny  | string  | the bitwise value of all disallowed permissions | 0       |
```

Comment:
Technically, both of these are strings, not numbers. Maybe it'd help to document them as such? (aka wrap them in "")

Reply:
I've changed them to strings, thanks.